src/adhocracy_frontend/adhocracy_frontend/static/js/ResourcesBase.ts

```diff
@@ -55,16 +53,4 @@ export class Resource implements IResource {
         this.data = {};
     }
     /* tslint:enable:variable-name */
-
-    public isInstanceOf(resourceType : string) : boolean {
-        var _class : IResourceClass = <any>this.constructor;
```

Comment:
(For my better understanding:) In general, this `IResourceClass` has been generated in MkResources.ts, right?